packages/welcome-screen/src/views/PackSelection.vue

```diff
@@ -16,18 +16,31 @@
           </b-col>
         </b-row>
       </b-container>
+      <p class="mb-5">
+        <UsbIcon />
+        <SdIcon />
+        If you have an Endless Key USB device or SD card,
+        <b-link to="endless-key">
+          click here
+        </b-link>
+        to get your pack!
```

Comment:
I don't have the design handy to compare, but this sort of `[text] <CLICK HERE> [text]` pattern can be problematic for users with screen readers or other accessibility aids. In particular, a screen reader is going to just call out "click here" when that link is selected unless we give it a title attribute, and either way it makes localization tricky. It would help if all of "click here to get your pack" was a link.

Granted, it's a small nitpick since we aren't targeting that use case at the moment, so if it turns into more work to make it look nice, it's fine for now as it is :)

Reply:
Totally agree! Also I think "press" is a better verb than "click" because it works for both desktop users and mobile/touchscreen users. I will change it in code and leave a comment in the design specs (tagging you).